### DISCUSSION TITLE
TV5

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#697] [Q&A]
**Channel Name:** TV5
**Channel ID:** TV5.ph
**Country:** Philippines
**Language:** Tagalog
**Website:** https://www.tv5.com.ph/
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by fotvnetwork** [comment]
TV5 livestream is on Cignal Play

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19716
